go-controller/pkg/util/util.go

```diff
@@ -362,6 +362,9 @@ func GetActiveNetworkForNamespace(namespace string, nadLister nadlister.NetworkA
 	for _, nad := range namespaceNADs {
 		nadInfo, err := ParseNADInfo(nad)
```

Comment:
I didn't even think about this non managed case.. I feel it should be inside parse nad right? when parsing nad's we should be ignoring these?

Reply:
To filter out NADs we can parse just type and then parse the whole thing in case if ovn type, I don't know if it's worth it, it has also a performance penalty.

Reply:
we parse all NADs now anyways.. :)